dotnet/src/SemanticKernel.Skills/Skills.UnitTests/OpenAPI/RestApiOperationTests.cs

```diff
@@ -0,0 +1,209 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Globalization;
+using System.Net.Http;
+using System.Web;
+using Microsoft.SemanticKernel.Skills.OpenAPI.Model;
+using Microsoft.SemanticKernel.Skills.OpenAPI.Rest;
+using Xunit;
+
+namespace SemanticKernel.Skills.UnitTests.OpenAPI;
+
+public class RestApiOperationTests
+{
+    [Fact]
+    public void ItShoudRenderHeaderValuesFormArguments()
```

Comment:
typo in `Should`, same in below methods. typo in `From`?

Reply:
Fixed